MatrixSDK/Utils/Logs/MXAnalyticsDestination.swift

```diff
@@ -30,12 +30,20 @@ class MXAnalyticsDestination: BaseDestination {
     
     override func send(_ level: SwiftyBeaver.Level, msg: String, thread: String, file: String, function: String, line: Int, context: Any? = nil) -> String? {
         let message = super.send(level, msg: msg, thread: thread, file: file, function: function, line: line, context: context)
-        #if !DEBUG
```

Comment:
Moving the compiler check to the parent, otherwise cannot write unit tests for this class